packages/near/src/token.ts

```diff
@@ -29,15 +29,18 @@ export async function create(
   // WARN: This is a non-standard JWT
   // Borrows ideas from: https://github.com/ethereum/EIPs/issues/1341
   const publicKey = await signer.getPublicKey(accountId, networkId);
-  const kid = encodeKey(publicKey.data);
+  const encoded = base58btc(publicKey.data);
```

Comment:
simplified big time, and the broker-core has received the corresponding update to the validation stuff.... but needs proper testing before we deploy